dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/JaxWsClientProxy.java

```diff
@@ -213,17 +238,28 @@ boolean isAsync(Method m) {
     }
 
     static SOAPFault createSoapFault(SOAPBinding binding, Exception ex) throws SOAPException {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         SOAPFault soapFault;
         try {
             soapFault = binding.getSOAPFactory().createFault();
+            if(isFinestEnabled)  {
+                LOG.finest("SOAPFault is created trough SOAPBinding SOAPFactory: " + soapFault);   // Liberty Change issue #26529
+            } 
         } catch (Throwable t) {
             //probably an old version of saaj or something that is not allowing createFault
             //method to work.  Try the saaj 1.2 method of doing this.
             try {
                 soapFault = binding.getMessageFactory().createMessage()
                     .getSOAPPart().getEnvelope().getBody().addFault();
+                if(isFinestEnabled)  {
+                    LOG.finest("SOAPFault is created trough SOAPBinding > message > SOAP part > Envelope > body: " + soapFault);   // Liberty Change issue #26529
```

Comment:
As a general comment, we should try to reduce the amount of trace that gets logged. I would suggest to just log useful debugging information without describing in detail what we are logging. For eg., I would change this trace to
 LOG.finest("SOAPFault created  using saaj 1.2 method: " + soapFault);